test/TestUtil.js

```diff
@@ -125,6 +125,37 @@ exports.markEnterprise = function (_this) {
     }
 };
 
+/**
+ * Duplicated this from BuildInfo because the logic of it is changed in 5.1, and in backward compatibility tests older
```

Comment:
Did we correct this in the actual code for 4.1.x? If so, we don't need this

Reply:
We don't need this method anyway. I thought I deleted it. Let me delete it.